README.md

```diff
@@ -53,7 +53,7 @@ Click "Save & Close" and ReachLocal Tracking will now be active on your site.
 
 ## License
 
-The ReachLocal Tracking Extension is licensed under the MIT license.
+The ReachLocal Tracking Extension is licensed under the GPLv2 license.
```

Comment:
Do we need to change the text below this line?

Reply:
Removed license text from the README as it is available in LICENSE.txt